### DISCUSSION TITLE
Allow for importing data from osu!stable without opening the first-run setup

### DISCUSSION THREAD

**Discussion by dotSepia** [ppy/osu#27766] [Ideas] [answered]
Self-explanatory, there's no way to import your data from osu!stable without going through the first-run setup so I thought this was a good idea. It's pretty annoying having to go through that and cancel it midway through just to import stuff from stable.

![image](https://github.com/ppy/osu/assets/103551093/c40222da-389d-438c-9dba-94ea550dabc2)

I suggest adding a button that opens a menu similar to this one in the setup wizard.

![image](https://github.com/ppy/osu/assets/103551093/50f85d8d-d2d3-4496-abad-88a2cba50c51)

It kind of weirded me out knowing that this wasn't already in the game. This seemed like a no-brainer to me so I can't tell why this isn't a thing but better late than never I guess. Maybe you can do this, but I couldn't find out how. Perhaps it is automatic, I don't know.

**Comment by Walavouchey** [comment, marked as answer]
this is intentional. the end goal is moving the playerbase over to lazer, meaning the first-run setup is the more correct place for the feature

**Reply by cjwrecks** [reply]
Yeah but as it is now most players are switching between both clients and it will be that way for most people for a good while regardless of how much better lazer is, it's silly that we can't just add an import button to the setting menu when it would make switching between lazer and back to stable easier for people and more user friendly. It's such a simple change that would make the experience better for so many people

**Reply by peppy** [reply]
If you search for import, the first run wizard will come up.

I know this isn't optimal, but to make it work well the dialog needs a revamp to allow opening it to a particular section (ie. hotlinking support).
